research-app/src/App.vue

```diff
@@ -26,14 +26,14 @@
           v-bind:imageset="imageset"
         />
       </div>
-      <div id="catalogs-container" v-if="haveCatalogs">
+      <div id="spreadsheets-container" v-if="haveTableLayers">
         <div class="display-section-header">
-          <label>Catalogs</label>
+          <label>Catalogs and Spreadsheets</label>
```

Comment:
So, terminology. The "spreadsheet" term is pretty WWT-specific — I think people will understand it, but I don't see a need to go out of our way to surface it. People will understand "catalog" to mean either their own data table, or something like a HiPS progressive catalog. So I think the name of this section can stay as just "Catalogs". Or "Data Tables" might be slightly more generic.

Then we should be more limited in where we use the "layer" terminology since that can include not just spreadsheet layers (both "small" and HiPS progressive), but also imagery layers like FITS files. (And the WWT engine provides some more obscure layer types that we don't currently use in these UIs.)